# target exposure
float32 target_exposure
---
# target value can not always be reached
float32 reached_exposure
bool success